epersons.md

```diff
@@ -45,6 +46,19 @@ the replace operation `[{ "op": "replace", "path": "/netid", "value": "newNetId"
   "netid": "newNetId",
 ```
 
+To replace the email value, `curl -X PATCH http://${dspace.url}/api/epersons/eperson/<:id-eperson> -H "Content-Type: application/json" -d '[{ "op": "replace", "path": "/email", "value": "new@email"]'`.  The operation also requires an Authorization header.
```

Comment:
Shouldn't the examples be updated to be using the endpoint?
`/api/eperson/epersons`